test/e2e/e2e_test.go

```diff
@@ -126,6 +142,24 @@ func checkConnectivityPingToHost(f *framework.Framework, nodeName, podName, host
 	if err != nil {
 		return err
 	}
+
+	// Wait for pod network setup to be almost ready
+	wait.PollImmediate(1*time.Second, 30*time.Second, func() (bool, error) {
+		pod, err := podClient.Get(podName, metav1.GetOptions{})
+		if err != nil {
+			return false, nil
+		}
+		if exGw {
+			_, ok := pod.Annotations["external-gw"]
```

Comment:
this is not the right annotation for external-gw?

Reply:
Yes you are right! I've fixed that and re-pushed.

Reply:
doesn't look different?

Reply:
ugh yeah sorry last commit got lost. pushed again